CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [1.2.17](https://github.com/ably/ably-cocoa/tree/1.2.17)
+
+[Full Changelog](https://github.com/ably/ably-cocoa/compare/1.2.16...1.2.17)
+
+**Fixed bugs:**
+
+- Ably `agents` client option is implemented incorrectly [\#1525](https://github.com/ably/ably-cocoa/issues/1525)
+
+**Closed issues:**
+
+- Implement API for fetching Agent identifier [\#1530](https://github.com/ably/ably-cocoa/issues/1530)
+- Fix error on trying to override `HTTPURLResponse._allHeaderFields` with Xcode 14 [\#1522](https://github.com/ably/ably-cocoa/issues/1522)
+- Replace usage of `SecKeychainItemDelete` to fix deprecation error in Xcode 14 [\#1521](https://github.com/ably/ably-cocoa/issues/1521)
+- The `make` script's `bump_patch` does not update all locations where version number exists [\#1447](https://github.com/ably/ably-cocoa/issues/1447)
```

Comment:
It's worth thinning out change log entries to remove lines like this, where they clearly carry no value to a user of the SDK.

Reply:
Sure thing, will do. Thanks!

Reply:
I'm going to resolve this one since it's a simple thing.